etc/api.report.api.md

```diff
@@ -183,29 +184,20 @@ export interface AsyncIterArgs<Q extends ObjectOrInterfaceDefinition, K extends
 // @public (undocumented)
 export interface Attachment {
     fetchContents(): Promise<Blob>;
+    // Warning: (ae-forgotten-export) The symbol "AttachmentMetadata" needs to be exported by the entry point index.d.ts
     fetchMetadata(): Promise<AttachmentMetadata>;
     // (undocumented)
     rid: string;
 }
 
-// @public (undocumented)
-export interface AttachmentMetadata {
```

Comment:
Do you think someone might want to use this type?

Reply:
Talked offline, going to withhold this type for now, we can easily add later if needed